vector/build.gradle

```diff
@@ -315,16 +318,9 @@ dependencies {
 
     def epoxy_version = '4.6.2'
     def fragment_version = '1.3.6'
-    def arrow_version = "0.8.2"
-    def markwon_version = '4.1.2'
     def big_image_viewer_version = '1.8.1'
     def glide_version = '4.12.0'
-    def moshi_version = '1.12.0'
-    def daggerVersion = '2.38.1'
     def autofill_version = "1.1.0"
-    def work_version = '2.5.0'
-    def arch_version = '2.1.0'
-    def lifecycle_version = '2.2.0'
     def rxbinding_version = '3.1.0'
     def jjwt_version = '0.11.2'
```

Comment:
Why not moving all the `def` values?

Reply:
Sure, it will be done in the next commit